provision/setup-machine-metrics

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
This seems worth testing. If we ever want to monitor these vms this way we'll need to expose the ports via the Vagrantfile but point them at different numbers...